pkg/routeagent/cleanuphandlers/cleanuphandlers.go

```diff
@@ -0,0 +1,6 @@
+package cleanuphandlers
+
+type CleanupHandler interface {
+	GetName() string
```

Comment:
I don't see this used anywhere ...

Reply:
Right, I wanted to use it on the error handling and forgot. Thanks